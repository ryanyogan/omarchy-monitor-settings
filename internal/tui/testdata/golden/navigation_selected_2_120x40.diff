Visual regression detected!

=== EXPECTED ===
  [90m╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                                                                [90m│[0m    
  [90m│[0m                                                [1;34mDisplay Settings[0m                                                [90m│[0m    
  [90m│[0m                                                                                                                [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                                                        
  [34m╭────────────────────────────────────────────╮[0m  [36m╭────────────────────────────────────────────────────────────────╮[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m                                                                [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m                                                                [36m│[0m    
  [34m│[0m  [1;34mNavigation[0m                                [34m│[0m  [36m│[0m  [1;36mDisplay Overview[0m                                              [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m                                                                [36m│[0m    
  [34m│[0m    [37mDashboard[0m                               [34m│[0m  [36m│[0m  [34m○[0m [1;32mHDMI-A-1[0m[1;33m 👆 CURRENT[0m                                         [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m  [37m  Dell U2414H[0m                                                 [36m│[0m    
  [34m│[0m    [37mMonitor Selection[0m                       [34m│[0m  [36m│[0m  [90m  1920x1080 @ 60Hz[0m                                            [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m  [90m  Scale: 1.0x[0m                                                 [36m│[0m    
  [34m│[0m  [1;32m▶[0m [1;32mSmart Scaling[0m                           [34m│[0m  [36m│[0m  [3;33m  → Scaling changes will apply here[0m                           [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m                                                                [36m│[0m    
  [34m│[0m    [37mManual Scaling[0m                          [34m│[0m  [36m│[0m  [90m◦[0m [1;34mDP-1[0m                                                        [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m  [37m  Samsung C27F390[0m                                             [36m│[0m    
  [34m│[0m    [37mSettings[0m                                [34m│[0m  [36m│[0m  [90m  1920x1080 @ 75Hz[0m                                            [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m  [90m  Scale: 1.2x[0m                                                 [36m│[0m    
  [34m│[0m    [37mHelp[0m                                    [34m│[0m  [36m│[0m                                                                [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m                                                                [36m│[0m    
  [34m│[0m    [37mExit[0m                                    [34m│[0m  [36m│[0m                                                                [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m                                                                [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m                                                                [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m                                                                [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m                                                                [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m                                                                [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m                                                                [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m                                                                [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m                                                                [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m                                                                [36m│[0m    
  [34m│[0m                                            [34m│[0m  [36m│[0m                                                                [36m│[0m    
  [34m╰────────────────────────────────────────────╯[0m  [36m╰────────────────────────────────────────────────────────────────╯[0m    
                                                                                                                        
  [90m╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                                                                [90m│[0m    
  [90m│[0m                           [1;32m↑↓[0m  [37mnavigate[0m    [1;34m⏎[0m  [37mselect[0m    [1;33mh[0m  [37mhelp[0m    [1;35mesc[0m  [37mback[0m    [1;31mq[0m  [37mquit[0m                         [90m│[0m    
  [90m│[0m                                                                                                                [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯[0m    

=== ACTUAL ===
  ╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                │    
  │                                                Display Settings                                                │    
  │                                                                                                                │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                                        
  ╭────────────────────────────────────────────╮  ╭────────────────────────────────────────────────────────────────╮    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │  Navigation                                │  │  Display Overview                                              │    
  │                                            │  │                                                                │    
  │    Dashboard                               │  │  ○ HDMI-A-1 👆 CURRENT                                         │    
  │                                            │  │    Dell U2414H                                                 │    
  │    Monitor Selection                       │  │    1920x1080 @ 60Hz                                            │    
  │                                            │  │    Scale: 1.0x                                                 │    
  │  ▶ Smart Scaling                           │  │    → Scaling changes will apply here                           │    
  │                                            │  │                                                                │    
  │    Manual Scaling                          │  │  ◦ DP-1                                                        │    
  │                                            │  │    Samsung C27F390                                             │    
  │    Settings                                │  │    1920x1080 @ 75Hz                                            │    
  │                                            │  │    Scale: 1.2x                                                 │    
  │    Help                                    │  │                                                                │    
  │                                            │  │                                                                │    
  │    Exit                                    │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  ╰────────────────────────────────────────────╯  ╰────────────────────────────────────────────────────────────────╯    
                                                                                                                        
  ╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                │    
  │                           ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit                         │    
  │                                                                                                                │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    

=== DIFF INFO ===
Expected length: 8053
Actual length: 6576
Expected hash: 2a59e082b503e334a1347c3662f80a3b1b65fb89419ce29e578c04e6dfe7ef1c
Actual hash: 6bafb854792ae21362b0568b0f34b004683c6890c34df768f264f585bc917e90
