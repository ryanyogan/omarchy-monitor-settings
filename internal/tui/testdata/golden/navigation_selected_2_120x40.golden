# Visual Golden File
# Name: navigation_selected_2
# Dimensions: 120x40
# Hash: 6bafb854792ae21362b0568b0f34b004683c6890c34df768f264f585bc917e90

  ╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                │    
  │                                                Display Settings                                                │    
  │                                                                                                                │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                                        
  ╭────────────────────────────────────────────╮  ╭────────────────────────────────────────────────────────────────╮    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │  Navigation                                │  │  Display Overview                                              │    
  │                                            │  │                                                                │    
  │    Dashboard                               │  │  ○ HDMI-A-1 👆 CURRENT                                         │    
  │                                            │  │    Dell U2414H                                                 │    
  │    Monitor Selection                       │  │    1920x1080 @ 60Hz                                            │    
  │                                            │  │    Scale: 1.0x                                                 │    
  │  ▶ Smart Scaling                           │  │    → Scaling changes will apply here                           │    
  │                                            │  │                                                                │    
  │    Manual Scaling                          │  │  ◦ DP-1                                                        │    
  │                                            │  │    Samsung C27F390                                             │    
  │    Settings                                │  │    1920x1080 @ 75Hz                                            │    
  │                                            │  │    Scale: 1.2x                                                 │    
  │    Help                                    │  │                                                                │    
  │                                            │  │                                                                │    
  │    Exit                                    │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  │                                            │  │                                                                │    
  ╰────────────────────────────────────────────╯  ╰────────────────────────────────────────────────────────────────╯    
                                                                                                                        
  ╭────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                │    
  │                           ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit                         │    
  │                                                                                                                │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    